config/git/hooks/post-checkout

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+command -v git-lfs >/dev/null 2>&1 || { echo >&2 "\nThis repository is configured for Git LFS but 'git-lfs' was not found on your path. If you no longer wish to use Git LFS, remove this hook by deleting '.git/hooks/post-checkout'.\n"; exit 2; }
```

Comment:
I think these comments have the wrong path - we install to `<root>/config/git/hooks/` but these reference `.git/hooks/`

Reply:
good point, we should fix this in our internal ones too heh

Reply:
Fixed!

Reply:
[773509a](https://github.com/slackhq/slack-gradle-plugin/pull/256/commits/773509a9fadf221e84f17cd6fbe68ea6d07c024f)